webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const nodeEnv = process.env.NODE_ENV || 'development';
+const isProd = (nodeEnv === 'production');
+
+module.exports = {
+  mode: nodeEnv,
+  optimization: {
+    minimize: isProd,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          compress:{
+            drop_console: true,
+          }
+        }
+      }),
+    ],
+  },
+  entry: {
+    dist: './h5p-editor-uuid.ts'
+  },
+  output: {
+    filename: 'h5p-editor-uuid.js',
+    path: path.resolve(__dirname, 'dist')
+  },
+  target: ['web', 'es5'], // IE11
```

Comment:
The `target` should be set more flexibly, in particular since IE11 is gone. It's helpful to set it to `['browserslist']` and to add something like `"browserslist": "> 0.5%, last 2 versions, Firefox ESR, not dead"` to `package.json` in order to dynamically create the target.